aa_py_xl_convert/_vbscript.py

```diff
@@ -1,9 +1,11 @@
 from pathlib import Path
 from subprocess import run
 
+from retry import retry
 from locate import this_dir
 
 
+@retry(tries=3, delay=2, backoff=2)
```

Comment:
Nice one! What backoff do? And how is a delay of 2?

Reply:
Read the docs!! https://pypi.org/project/retry/

> @retry((ValueError, TypeError), delay=1, backoff=2)
def make_trouble():
    '''Retry on ValueError or TypeError, sleep 1, 2, 4, 8, ... seconds between attempts.'''